src/lib/InstantSearch.ts

```diff
@@ -137,6 +145,7 @@ class InstantSearch extends EventEmitter {
   public _searchStalledTimer: any;
   public _isSearchStalled: boolean;
   public _searchParameters: PlainSearchParameters;
+  public _initialUiState: UiState;
```

Comment:
would  be nice if we could avoid storing it, but since we need it in `start`, not in the constructor, it isn't possible